### ISSUE TITLE
[BUG] Native Window does not close.

### ISSUE THREAD

**Issue by mathewodwyer** [opentk/opentk#1382]
### Description

When creating a new project, I create a native window and run my application. For some reason when I press the client button the window does not close. The window is responsive but it does not close.

### Repro steps

1. Create a new .NET 5.0 project
2. Install _OpenTK.Windowing.Desktop 4.6.7_ from NuGet.
3. Copy and paste the following code into the main method.

```csharp
using (var nativeWindow = new NativeWindow(NativeWindowSettings.Default))
{
	while (!nativeWindow.IsExiting)
	{
		nativeWindow.ProcessEvents();
	}
}
```

4. Run the application to see the window does not close.

### Expected behavior

The window should close when you press the close button.

### Actual behavior

The window does not close when I press the close button.

https://user-images.githubusercontent.com/95602085/152624676-657460da-f8d9-426b-935d-88042651da30.mp4

### Related information 

* Operating system: Windows 10 Enterprise
* What version of OpenTK: NuGet 4.6.7
* What runtime: .NET 5.0
* Workarounds (if any): None that I can think of.

Happy to assist in any way I can to resolve this issue.

**Comment by tottaka** [comment]
Easiest workaround I could figure out immediately, not optimal but seems to work:
`class Program
{
	static bool IsRunning = true;

	static void Main(string[] args)
	{
		using (var nativeWindow = new NativeWindow(NativeWindowSettings.Default))
		{
			nativeWindow.Closing += NativeWindow_Closing;

			while (IsRunning)
			{
				nativeWindow.ProcessEvents();
			}
		}
	}

	private static void NativeWindow_Closing(System.ComponentModel.CancelEventArgs obj)
	{
		if (!obj.Cancel)
			IsRunning = false;
	}
}`

**Comment by NogginBops** [comment]
I am unable to repro this issue on my machine.
Running the code you posted above the window closes as it should and there is no issue.
This might have something to do with me testing it on a windows 7 laptop, I can try repro this later on my Windows 10 machine.

**Comment by NogginBops** [comment]
Since I can't repro this I'll move this to the `4.7.1` milestone so that we can get ´4.7.0` released.

**Comment by NogginBops** [comment]
Turns out this issue was fixed in #1338 and I was trying to repro this with a version that had that PR merged.
